lib/ui/Popup.js

```diff
@@ -237,7 +238,7 @@ Popup.prototype = {
 
     // searchTerm is the same thing as the 'text' item sent down in each result.
     // maybe that's not a useful place to put the search term...
-    let searchTerm = controller.searchString.replace(/^\s+/, "").replace(/\s+$/, "")
+    let searchTerm = controller.searchString.replace(/^\s+/, '').replace(/\s+$/, '');
```

Comment:
Same as [above](https://github.com/mozilla/universal-search-addon/pull/39/files#r34214814). Can we just use `controller.searchString.trim()`?